lib/services/google_play_api.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+Future<Map<String, dynamic>> getPackageInfo(String pkgName) async {
+  String result = '{}';
+  if (pkgName == 'com.vanced.android.youtube') {
+    result =
+        '{"name":"YouTube Vanced","image":"https://downloadr2.apkmirror.com/wp-content/uploads/2019/02/youtube-vanced.jpg", "pkgName":"com.vanced.android.youtube"}';
+  } else {
```

Comment:
Before hitting any additional IP a warning has to be displayed to the user. This also goes for the sources or API links. Additionally this implementation forces using Google Play Store. What if I don't want to use Google Play Store? This should be modularized enough, so that the provider for app names and icons can be switched out through an interface.

Reply:
If you design an interface, you can implement both or one of them. The important part is that the interface exists so if necessary the provider can be switched out.

Reply:
OK. So, can I close this pull and create a new one after adding the requested changes?

Reply:
> OK. So, can I close this pull and create a new one after adding the requested changes?

It will be alot of effort. I do not think it is worth spending time on. Current manager will replaced by Compose rewrite where this is already in the works.